pkg/cmd/cli/cmd/rollout/rollout.go

```diff
@@ -134,3 +135,20 @@ func NewCmdRolloutUndo(fullName string, f *clientcmd.Factory, out io.Writer) *co
 	cmd.Example = fmt.Sprintf(rolloutUndoExample, fullName)
 	return cmd
 }
+
+const (
+	rolloutStatusLong = `
+Watch the status of the latest rollout, until it's done.`
```

Comment:
This proposal needs to move upstream

On Wed, Jul 20, 2016 at 7:50 PM, Fabiano Franz <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/rollout/rollout.go
> <https://github.com/openshift/origin/pull/9874#discussion_r71573627>:
>
> > @@ -134,3 +135,20 @@ func NewCmdRolloutUndo(fullName string, f *clientcmd.Factory, out io.Writer) *co
> >  	cmd.Example = fmt.Sprintf(rolloutUndoExample, fullName)
> >  	return cmd
> >  }
> > +
> > +const (
> > +	rolloutStatusLong = `
> > +Watch the status of the latest rollout, until it's done.`
>
> Meaning the command waits and blocks and only exit when the rollout is
> completed? In such case I'd vote for oc rollout watch or oc rollout
> status --watch (exits right away without the flag) for consistency.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9874/files/7384f4c3e74a711cdb73ecb1d1958c1b3dc6f251#r71573627>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADuFf3RedOQQ1cZeVl6Rb8Otb6xPqxmrks5qXl_3gaJpZM4JNVe5>
> .
>